pymysql/connections.py

```diff
@@ -374,6 +396,9 @@ def is_auth_switch_request(self):
         # http://dev.mysql.com/doc/internals/en/connection-phase-packets.html#packet-Protocol::AuthSwitchRequest
         return self._data[0:1] == b'\xfe'
 
+    def is_rsa_public_key(self):
+        return self._data[0:1] == b'\x01'
```

Comment:
I found the real meaning of `\x01`, it means  [Extra Authentication Data](https://web.archive.org/web/20170404163312/https://dev.mysql.com/doc/internals/en/successful-authentication.html).